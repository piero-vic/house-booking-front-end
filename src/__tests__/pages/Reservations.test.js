import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/store';
import Reservations from '../../pages/reservations';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <Reservations />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
