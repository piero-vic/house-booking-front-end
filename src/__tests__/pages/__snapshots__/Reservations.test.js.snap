// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container w-4/5 mx-auto h-screen pt-20 sm:pt-32"
>
  <div
    className="mb-5"
  >
    <h2
      className="text-center text-black font-black text-3xl"
    >
      RESERVATIONS
    </h2>
    <p
      className="text-center text-gray-500 font-semibold text-1xl"
    >
      Take a look at your reservations
    </p>
  </div>
  <div
    className="sc-dmRaPn gelpCx"
  >
    <div
      className="sc-fLlhyt ifOHjV"
    >
      <div
        className="sc-bczRLJ eSTlnH rdt_Table"
        disabled={false}
        role="table"
      >
        <div
          className="sc-ivTmOn fwKvpK"
        >
          <div
            style={
              Object {
                "padding": "24px",
              }
            }
          >
            There are no records to display
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
